Modules/Control/Field_To_Current.py

```diff
@@ -0,0 +1,9 @@
+import math
+AMOUNT_OF_WRAPS = 100
```

Comment:
Do you want to export all your constants and settings to a separate config file?


(Hint: the answer is "yes", you want to)

Reply:
Just like in the rest of the modules.